env-var-docs/generate_markdown.py

```diff
@@ -171,7 +178,7 @@ def append_node_to_out(node: env_var_docs.parser.Node):
             if var.values is not None:
                 out += "- Allowed values:\n"
                 for val in var.values:
-                    out += f"   - {val}\n"
+                    out += f"   - `{val}`\n"
```

Comment:
question (non-blocking): just curious why we need the backticks. was it not printing correctly?

Reply:
It prints fine without them, but the backticks format the values as code with a monospace font, making them a little easier to read to ensure each character gets copied correctly if someone copies and pastes a value

Reply:
nice!